pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -136,14 +136,13 @@ def _call_children_scripts(self):
         if self.parallel_devices is None:
             raise MisconfigurationException("you selected (distribute_backend = ddp) but did not set Trainer(gpus=?)")
 
-        os.environ["PL_TRAINER_GPUS"] = ",".join([str(device.index) for device in self.parallel_devices])
+        # os.environ["PL_TRAINER_GPUS"] = ",".join([str(device.index) for device in self.parallel_devices])
```

Comment:
Why is it commented ?

Reply:
That's the fix and this PR is WIP

need to debug more why tests are passing locally and not in CI